### PR TITLE
Remove some useless drugs

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

See CL

## Why It's Good For The Game

Cleaning some useless drugs.

Unlike others, ultrazine is usefull in some cases but having marine being speed demons for a zero cost is not welcomed either

## Changelog
:cl:
delete: Remove quietus, somnolent, stimulum, ultrazine, rezadone, roulettium, para, arithrazine
/:cl:

### CODE REVIEW

**Comment by Adven4er** [comment]
as far as I know, only some ERTs have "ultra", what is the point of deleting it under the pretext of "marinas are too fast for zero cost".

stimulon is not bad, but looking at his recipe, I’m not surprised that they don’t use it.

ruletium is good for "last chance" but no more, and I don't think many people know about it due to the lack of any information on the wiki (which can be said about rezadon)

P.s. And what is "somnolent"?

**Comment by FosterGrime-Heymynameslouie** [comment]
saw title and the first thing that went to mind is "paracetamol" 

honestly what purpose does it serve apart from it being a meme? I might be dumb or stupid or maybe both, and I only seen it been used once in all of my playtime.

**Comment by Losenis** [comment]
>honestly what purpose does it serve apart from it being a meme?
Very long term premed that doesn't purge ryetalin and has the same metab rate, you can combine it with bandages and ointments that now are viable and you can just forget about meds with an almost passive healing at the cost of not being a turbopremeded dunga that can pretend he's doomguy.

**Comment by BraveMole** [comment]
> saw title and the first thing that went to mind is "paracetamol"
> 
> honestly what purpose does it serve apart from it being a meme? I might be dumb or stupid or maybe both, and I only seen it been used once in all of my playtime.

It has super niche use. Clearly it's on a lifeline but I don't want to remove rn

**Comment by Arctrooper8** [comment]
The vali has a specific use for para via the suit injection bonus. Could you replace it with some other chem if you're removing Para?

You also might have messed up the ryetalin recipe by removing Arithrazine, because right now the recipe is simply carbon.

**Comment by BraveMole** [comment]
> The vali has a specific use for para via the suit injection bonus. Could you replace it with some other chem if you're removing Para?

Uh, i can think of something I guess

> You also might have messed up the ryetalin recipe by removing Arithrazine, because right now the recipe is simply carbon.

Nop, it's just that rye recipe is changed in other pr

**Comment by SzkieletorNutric** [comment]
Stimulon is never used not because it's useless but because its recipe pretty much off-limits unless both Researchers give up their 10u bottles of Lemoline, in exchange for a niche super-drug.

Same goes for any other drug that requires a large amount of Lemoline as an ingredient. People rarely bother/have the time to even gather up the extracted alien embryos and duplicate the Lemo with them.

Not Somnolent - Somolent.
One of the few Lemoline-based drugs that can actually be brewed roundstart, how the hell do you consider this useless?..

Gradual cloneloss-healing and general quick recovery while unconcious. A sleeping drug to compliment healing and/or groundside surgery while in safety. Not that overpowered either, as it certainly can't replace the cryotubes when it comes to large amounts of cloneloss.

If anything this is the one drug that ought to stay

File: code/modules/reagents/reactions/medical.dm [hunk 1]
```diff
@@ -38,7 +38,7 @@
 /datum/chemical_reaction/ryetalyn
 	name = "Ryetalyn"
 	results = list(/datum/reagent/medicine/ryetalyn = 2)
-	required_reagents = list(/datum/reagent/medicine/arithrazine = 1, /datum/reagent/carbon = 1)
+	required_reagents = list(/datum/reagent/carbon = 1)
```
Comment (QualityVan) [inline code/modules/reagents/reactions/medical.dm:41]: I know this is handled in another pr you have up but still.

File: code/modules/reagents/reactions/other.dm [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /datum/chemical_reaction/serotrotium
 	name = "Serotrotium"
 	results = list(/datum/reagent/serotrotium = 1) //Weird emotes, chance of minor drowsiness.
-	required_reagents = list(/datum/reagent/medicine/paracetamol = 1, /datum/reagent/medicine/tramadol = 1)
+	required_reagents = list(/datum/reagent/medicine/tramadol = 1)
```
Comment (QualityVan) [inline code/modules/reagents/reactions/other.dm:4]: Yeah

**Comment by QualityVan** [comment]
I think on the whole this is overaggressive. Paracetamol has a decent dedicated use without being overpowering (aside from the pain stuff, which as ever is questionably tuned). Arithrazine, quietus, and rezadone definitely don't particularly need to stick around. Roulettium is, uh. Yeah. Stimulum and ultrazine fall into the problem category of speed drugs being really, really good regardless of drawbacks. I thought one of the ERTs used ultra, but maybe that's hyper? Somnolent is fine as a lemoline product, it just needs its name changed to add the missing N.

**Comment by Frinid1** [comment]
Buff para by removing purge from dylo, para literally have zero connection with toxin chemistry, but for some reason they purge

**Comment by zzurge** [comment]
coders just really be like "lets remove all the old stuff you love and know just because"

**Review by TiviPlus — CHANGES_REQUESTED** [review]
imo shouldnt some uses of para be replaced with trama (example on map or ryetalyn recepie

rest lgtm

**Comment by disappointedButNotSuprised** [comment]
I would advise against removig rezadone since it is used in the special admin spawn elite autoinjector that are featured in the deathsquad medkits

**Pull request closed without merging**